companion/src/CMakeLists.txt

```diff
@@ -373,21 +449,11 @@ IF(APPLE)
   # Use a non standard Info.plist that adds Retina support flags
   set_target_properties(${COMPANION_NAME} PROPERTIES MACOSX_BUNDLE_INFO_PLIST ${PROJECT_SOURCE_DIR}/companion/targets/mac/MacOSXBundleInfo.plist.in)
   set_target_properties(${SIMULATOR_NAME} PROPERTIES MACOSX_BUNDLE_INFO_PLIST ${PROJECT_SOURCE_DIR}/companion/targets/mac/MacOSXBundleInfo.plist.in)
-ENDIF(APPLE)
-IF(WIN32)
-  set(APPS "\${CMAKE_INSTALL_PREFIX}/bin/QtTest.exe")
-ENDIF(WIN32)
-
 
-INSTALL(TARGETS ${COMPANION_NAME}
-    BUNDLE DESTINATION . COMPONENT Runtime
-    RUNTIME DESTINATION bin COMPONENT Runtime
-    )
-
-# Qt5 + Cmake + Mac is poorly documented. A lot of this is guesswork
-# and trial and error. Do not hesitate to fix it for the better
-if(APPLE)
-  set(APPS "\${CMAKE_INSTALL_PREFIX}/${COMPANION_NAME}.app")
```

Comment:
This line was redundant, look up [here](#diff-a1ea07d95d9868fee378f5c869d8d448L369) (lines left:369 right:445)

There were a few other redundancies in the OSX script as well.